### PR TITLE
Create a dedicated repo for XOR tries.

### PR DESCRIPTION
**Pull request by petar**
This commit creates the repo. Make sure go.mod and LICENSE meet PL standards.
The initial code is already checked in master.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Protocol Labs, Inc.
```
Comment (willscott) [inline LICENSE]: 2020 :)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+> Implementation of XOR tries
+
+## Summary
+
+This package implements XOR tries.
+An XOR trie is a trie for equal-length bit strings.
+XOR tries support efficient set operations, as well as distance-based operations with respect to the XOR metric.
+
+## License
```
Comment (willscott) [inline README.md]: look at the other libp2p repos. probably worth adding the standard 'Contributing' section, and eventually usage / etc.

File: LICENSE (hunk 1 above)
Comment (petar) [reply LICENSE]: Done.

File: README.md (hunk 2 above)
Comment (petar) [reply README.md]: Done.

**Pull request merged**